### PR TITLE
Include container mapping csv file and adjust tests.

### PR DESCRIPTION
**Pull request by UlysseFG**

### CODE REVIEW

**Comment by nafiz1001** [comment]
```
=================================== FAILURES ===================================
__________________ SamplePoolingplanningTestCase.test_import ___________________

self = <fms_core.tests.test_template_importers.sample_pooling_planning_test.SamplePoolingplanningTestCase testMethod=test_import>

    def test_import(self):
        for file in self.files:
            # Basic test for all templates - checks that template is valid
            result = load_template(importer=self.importer, file=file)
>           self.assertEqual(result["valid"], True)
E           AssertionError: False != True

fms_core/tests/test_template_importers/sample_pooling_planning_test.py:103: AssertionError
```

**Comment by nafiz1001** [comment]
> ```
> =================================== FAILURES ===================================
> __________________ SamplePoolingplanningTestCase.test_import ___________________
> 
> self = <fms_core.tests.test_template_importers.sample_pooling_planning_test.SamplePoolingplanningTestCase testMethod=test_import>
> 
>     def test_import(self):
>         for file in self.files:
>             # Basic test for all templates - checks that template is valid
>             result = load_template(importer=self.importer, file=file)
> >           self.assertEqual(result["valid"], True)
> E           AssertionError: False != True
> 
> fms_core/tests/test_template_importers/sample_pooling_planning_test.py:103: AssertionError
> ```
> 
> ```
> (env) nislam@Laelia:~/projects/freezeman/backend$ git status
> On branch pooling-container-mapping
> Your branch is up to date with 'origin/pooling-container-mapping'.
> ```

Turns out, I just forgot to run `python manage.py collectstatic`

**Comment by nafiz1001** [comment]
```python

                for filename in content_zipped.namelist():
                    if filename[-5:] != ".xlsx":  # At the moment we only test the csv output files.
                        csv_content = {}
```

Wouldn't it be better/safer to test if it's equal to csv instead?

File: backend/fms_core/template_importer/importers/sample_pooling_planning.py
[the remark this answers is not in the document]
Comment (UlysseFG) [reply backend/fms_core/template_importer/importers/sample_pooling_planning.py:1]: BytesIO is not a great type to manipulate data. I like to keep my options open until the last minute when the data is ready. The performance is really not an issue here.

**Comment by UlysseFG** [comment]
testing for csv would be pretty much the same.

File: backend/fms_core/tests/test_containers.py [hunk 1]
```diff
@@ -19,4 +19,4 @@ def test_container_spec(self):
             "is_run_container": False,
         })
 
-        self.assertTrue(CONTAINER_SPEC_ROOM.is_source)
+        self.assertTrue(CONTAINER_SPEC_SITE.is_source)
```
Comment (nafiz1001) [inline backend/fms_core/tests/test_containers.py:22]:
```
FAILED fms_core/tests/test_containers.py::AdminUtilsTestCase::test_container_spec - AssertionError: False is not true
```

Comment (nafiz1001) [reply backend/fms_core/tests/test_containers.py:22]: Although I'm not sure if it's that line that is throwing the error.

Comment (nafiz1001) [reply backend/fms_core/tests/test_containers.py:22]: Never mind.

**Review by nafiz1001 — APPROVED** [review]

**Pull request merged**